stdlib/public/SwiftShims/CMakeLists.txt

```diff
@@ -79,7 +79,20 @@ else() # NOT SWIFT_BUILT_STANDALONE
   set(clang_headers_location "${LLVM_LIBRARY_DIR}/clang/${CLANG_VERSION}")
 endif()
 
-add_custom_command_target(unused_var
+if("${CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
+  add_custom_command_target(unused_var
+    COMMAND
+      "${CMAKE_COMMAND}" "-E" "make_directory" "${SWIFTLIB_DIR}"
+    COMMAND
+      "mklink" "/J"
```

Comment:
In LLVM our normal approach is to replace "create_symlink" with "copy" on non-unix systems. I think that is way simpler, and a better approach. Especially because calling `mklink` on Windows requires elevated privileges, and I think it would be unfortunate if building Swift required elevated privileges. That would specifically hamper the ability of students to checkout and build Swift on University-owned machines.